src/slic3r/GUI/PrintHostDialogs.cpp

```diff
@@ -214,7 +224,7 @@ void PrintHostSendDialog::EndModal(int ret)
             path = path.SubString(0, last_slash);
                 
 		AppConfig *app_config = wxGetApp().app_config;
-		app_config->set("recent", CONFIG_KEY_PATH, into_u8(path));
+		app_config->set("recent", CONFIG_KEY_PATH, into_u8(m_save_recent_path ? path : ""));
```

Comment:
I have some concerns that we are making thing too complex.
I would suggest we just support forward slash defined in `filename_format`, so that it works for both export and upload logic

Reply:
commented on in above conversation